Clarifications, more active voice, American spellings (#153)
thegooddocsproject/templates: how-to/about-how-to.md

How-to articles take the reader through a series of steps required to solve a specific, real-world problem.
Think of them as recipes for getting stuff done!

Sometimes you need to explain in detail how to do a specific task. Instead of teaching a concept, you are providing a set of sequential
steps that users must follow in order.

How-to articles are often confused with tutorials, but there is a difference. How-to articles are _problem_-oriented, not _learning_-oriented.

How-to articles are used by those who have experience with the product and need a specific question answered.
New users who want to solve a problem immediately can also benefit from how-to articles, provided they are well-written and clearly state any prerequisite knowledge required.

## Content of your how-to article

### About the "Overview" section

Summarize what the reader will get from reading the how-to articles.

> Creating a good overview comes down to thinking about will use the doc:

Reviewer: ```suggestion
Creating a good overview comes down to thinking about who will be reading the document:
```

* Are you writing only for end-users? For developers?
* Are you writing only for people who have a certain problem to solve?
* Is it intended for a particular industry?

### About the "Before you start" section

Have you ever gotten halfway through a how-to, only to discover that you need to read other documentation before you can continue?
This section's purpose is to prevent that from happening.

> Describe what the audience needs to know or have before following this how-to. Prerequisites can include other articles or information
> the reader needs to understand first. This list can also include technical dependencies, such as requiring an internet connection.
> By stating the requirements up-front, you prevent your audience from having a bad experience with your how-to.
> Whenever possible, include links to procedures or information, or give your audience tips about how to get whatever they need.

Reviewer: ```suggestion
Describe what the reader needs to know or have before following this how-to. Prerequisites can include other articles or information the reader needs to understand first. This list can also include technical dependencies, such as requiring an internet connection. By stating the requirements up-front, you prevent your audience from having a bad experience with your how-to. Whenever possible, include links to procedures or information, or give your audience tips about how to get whatever they need.
```

Reviewer: Not entirely certain what's happening with the line breaks here.

* Access to the Postman application.
* An understanding about what a RESTful API is conceptually. For more information about RESTful APIs, see http://example.com/restful_apis. 
* (Optional) A development tool (IDE) that displays API responses formatted for readability.

> * You need to have a list of favorites before you can manage them. For more information about favourites lists, see #Create_Favorites.

Reviewer 2: We still need to change to the American spelling for "favorites" in the last instance at the end of this sentence.
